### PR TITLE
download_gtfs.py: `feed["spec"]` is not "GTFS"

### PR DESCRIPTION
**Pull request by yellowhat**
While using the download_gtfs.py script seems that feed["spec"] values is GTFS not gtfs.

Signed-off-by: Yellowhat <yellowhat46@gmail.com>

### CODE REVIEW

File: tools/python/transit/gtfs/download_gtfs.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def get_feeds_links(data):
     gtfs_feeds_urls = []
 
     for feed in data:
-        if feed["spec"] != "gtfs":
+        if feed["spec"].lower() != "gtfs":
```
Comment (biodranik) [inline tools/python/transit/gtfs/download_gtfs.py:63]: Does it make sense to print a warning in this case? What are other possible values here?

Comment (yellowhat) [reply tools/python/transit/gtfs/download_gtfs.py:63]: From what I can see:
- `MDS`
- `GBFS`
- `GTFS_RT`
- `GTFS`

I have no idea what is the difference

Comment (biodranik) [reply tools/python/transit/gtfs/download_gtfs.py:63]: Can you please add this list to the comments in this switch? These are different types of feeds.

Comment (yellowhat) [reply tools/python/transit/gtfs/download_gtfs.py:63]: If there has been a switch from `gtfs` to `GTFS`, we should just change to `if feed["spec"] != "GTFS"`.
I have no ideas what the other `spec` means.

Comment (biodranik) [inline tools/python/transit/gtfs/download_gtfs.py:63]:
```suggestion
        # Possible values: MDS, GBFS, GTFS_RT, GTFS
        if feed["spec"].lower() != "gtfs":
```

**Comment by yellowhat** [comment]
Done

**Review by biodranik — APPROVED** [review]

**Pull request merged**